starknet_py/tests/e2e/account/account_test.py

```diff
@@ -114,10 +118,14 @@ async def test_account_estimate_fee_for_declare_transaction(
     account, map_compiled_contract_and_class_hash
 ):
     (compiled_contract, class_hash) = map_compiled_contract_and_class_hash
+    resource_bounds = ResourceBoundsMapping(
+        l1_gas=MAX_RESOURCE_BOUNDS_L1,
+        l2_gas=ResourceBounds.init_with_zeros(),
+    )
```

Comment:
It can be a constant

Reply:
Maybe we can set` l1_gas` and `l2_gas` as optional in `ResourceBoundsMapping` and set them to zero if they are not passed?

Reply:
Good catch 👍 , updated.